LeVent.Tests.Unit/Services/Processings/Events/EventProcessingServiceTests.Exceptions.Add.cs

```diff
@@ -19,7 +18,7 @@ public partial class EventProcessingServiceTests
     {
         [Theory]
         [MemberData(nameof(DependencyValidationExceptions))]
-        public void ShouldThrowDependencyValidationExceptionOnAddIfDependencyValidationErrorOccurs(
+        private void ShouldThrowDependencyValidationExceptionOnAddIfDependencyValidationErrorOccurs(
```

Comment:
Test methods must be public. The xUnit framework identifies test methods based on their public access modifier and the absence of parameters.

```suggestion
        public void ShouldThrowDependencyValidationExceptionOnAddIfDependencyValidationErrorOccurs(
```

Reply:
In and around the time of this video (link below) regarding the Standard version upgrade of 2.10.0 and as well subsequent videos prior to or post of the video. It was stated that the XUnit test methods can be made private. Although the clsss must remain public.

I did not arbitrarily decide to make them private on my own accord, based on my opinion. But to share my opinion marking them private has no ill effect on running the tests. Please inform on why they MUST be public. I did some research and found some threads stating the same as I have above and as was in the videos by Hassan.

https://youtu.be/oyJvszELGwE?si=NlbZMYYqhsgd5Jay
https://github.com/xunit/xunit/discussions/2438
https://stackoverflow.com/questions/16214684/why-is-the-xunit-runner-not-finding-my-tests/16214704#16214704
https://xunit.net/xunit.analyzers/rules/